app/metrics.yaml

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+legacy.ids:
+  client_id:
+    type: uuid
+    lifetime: user
+    description: |
+      The client id from legacy telemetry.
+    send_in_pings:
+      - deletion_request
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1635427
```

Comment:
This is currently pointing to the Firefox-iOS bug/review, please don't forget to update it.  You can even point it at this PR, and we can request review in PR comments to keep everything in once place.